test/specs/calculatorTest.spec.js

```diff
@@ -0,0 +1,21 @@
+const { expect } = require("chai");
```

Comment:
Please use **or** only single-quotes **or** only double-quotes in entire project. Not both of them.